tests/ast/data/DataFrame.to_local_iterator.test

```diff
@@ -0,0 +1,32 @@
+## TEST CASE
+
+df = session.table("test_table")
```

Comment:
This is the relevant test for the PR, the rest of the tests are updating the base64 string generated.